Sources/ScipioKit/Producer/PIF/BuildParametersGenerator.swift

```diff
@@ -3,8 +3,8 @@ import TSCBasic
 import XCBuildSupport
 import SPMBuildCore
 
-struct XCBBuildParameters: Encodable {
-    struct RunDestination: Encodable {
+struct XCBBuildParameters: Encodable, Sendable {
```

Comment:
Just a question: what was the reason for adding `Sendable`?

Reply:
I don't remember the details, but it's used somewhere.

This struct is not public and struct. So it may be `Sendable` implicitly, so we might not specify this protocol.